sdk/purview/azure-analytics-purview-scanning/swagger/README.md

```diff
@@ -21,4 +21,5 @@ add-context-parameter: true
 context-client-method-parameter: true
 generate-sync-async-clients: true
 generate-samples: true
+generate-builder-per-client: false
```

Comment:
we need 
`enable-sync-stack: true`

how did we regen without that?

Reply:
It be default in later version of codegen https://github.com/Azure/autorest.java/blob/main/preprocessor/data-plane.md
Hence unless dev sets `enable-sync-stack: fasle`, DPG will generate sync stack.

Yeah, then I kind of not know why devcenter generates one without sync-stack.

Reply:
><-- sorry, I was mistaken, devcenter is already sync-stack

devcenter is not on sync stack

Reply:
I mean their ongoing PR for next release https://github.com/Azure/azure-sdk-for-java/pull/34918